src/main/scala/dpla/ingestion3/model/package.scala

```diff
@@ -128,7 +138,9 @@ package object model {
            */
           // ("specType" -> "FIXME") ~
           // stateLocatedIn is omitted.
-          ("subject" -> record.sourceResource.subject.map{s => s.providedLabel}) ~
+          ("subject" -> record.sourceResource.subject.map{s =>
```

Comment:
Surprised anything hit the index before this without this fix.

Reply:
They'll hit the index but it won't render in the CQA app